plugins/logger/frontend/public/javascripts/countly.views.js

```diff
@@ -369,6 +370,28 @@ window.LoggerView = countlyView.extend({
             currentLogItem = currentLogItem[0];
             $("#logger-selector").clySelectSetSelection(currentLogItem.value, currentLogItem.name);
         }
+    },
+    renderPageWarningAlert: function(message) {
+        $(".routename-logger #logger-alert").html("");
+        var warningAlertContainer = '<div class="event-remind-tooltip" style="display: block;">' +
+                    '<div class="content">' +
+                            '<span class="prefix"></span>' +
+                            '<span class="remind-context">' + message + '</span>' +
+                            '<i class="fas fa-times"></i>' +
+                    '</div>' +
+                '</div>';
+        setTimeout(function() {
+            $(".routename-logger #logger-alert").append(warningAlertContainer);
+            $(".event-remind-tooltip .fa-times").off("click").on("click", function(e) {
+                $(e.currentTarget.parentElement.parentElement).remove();
+            });
+        }, 0);
+    },
+    displayLoggerStateWarningAlertIfNecessary: function() {
+        var loggerRequestAppState = countlyGlobal.apps[countlyCommon.ACTIVE_APP_ID].plugins.logger.state;
```

Comment:
Hello,
I think if the plugin does not have any separate configs changed, it is possible the app document would not have plugins property at all. So all descending properties should be null checked.